src/Forms/Prism.Forms/Navigation/NavigationException.cs

```diff
@@ -29,4 +30,14 @@ public NavigationException(string message, Page page, Exception innerException)
 
         public Page Page { get; }
     }
+
+    public class AmbiguousPathException : Exception
+    {
+        public string Name { get; }
+
+        public AmbiguousPathException(string name)
+        {
+            Name = name;
+        }
+    }
```

Comment:
this should be moved to it's own file and used as the inner exception to a NavigationException for the appropriate use case